conntable.go

```diff
@@ -71,20 +74,42 @@ func (t *connTable) Register(ck ckey, matchedRule *Rule, srcIP, srcPort string,
 			TargetPort: targetPort,
 			//TargetIP:   targetIP,
 		}
+
 	}
 }
 
-func (t *connTable) FlushOlderThan(s time.Duration) {
+func (t *connTable) FlushOlderOnes() {
 	t.mtx.Lock()
 	defer t.mtx.Unlock()
 
-	threshold := time.Now().Add(-1 * s)
+	for minutes := 5; minutes >= 0; minutes -= 1 {
```

Comment:
In replace ```30``` with ````5````  for testing and forgot to remove. Well my logic was actually:
`for minutes := 30; minutes >= 0; minutes -= 10` 
I though `30` minutes are enough to have `1K` connections. so its ok for temporary work around?